src/App.jsx

```diff
@@ -1,60 +1,281 @@
-import "./App.css"
+import { useState } from "react";
+
+import "./App.css";
 
 function App() {
+  const [operator, setOperator] = useState("+");
+  const [FirstNumber, setFirstNumber] = useState(0);
+  const [SecondNumber, setSecondNumber] = useState(0);
+  const [Result, setResult] = useState(0);
+  const [StoredValue, setStoredValue] = useState(0);
+
+  function calculate(num1, num2, operator) {
```

Comment:
rename to calculateTotal for more clarity

also, no need to include `num1, num2, ... ` as parameters since those values are already available to you from the state variables above